app/src/main/java/com/example/habitsmasher/ui/today/TodayFragment.java

```diff
@@ -227,7 +226,11 @@ public void updateListAfterEdit(Object editedObject, int pos) {
         //not needed
     }
 
-    // helper method to make a habit
+    /**
+     * Helper method for making habits
+     * @param extractMap the information to make a habit
+     * @return new habit
+     */
```

Comment:
Turned all of the header comments into javadocs as per @rudy-patel's request.

Reply:
Love it, thanks man